octoprint_mrbeam/__init__.py

```diff
@@ -1335,16 +1341,20 @@ def _getCurrentFile(self):
 			return None, None
 
 	def getHostname(self):
-		'''
-		Get device hostnema like 'MrBeam2-F930'
-		:return: String hostname or empty string
-		'''
-		hostname = '';
+		"""
+		Returns device hostname like 'MrBeam2-F930'.
+		ALSO: Checks if hostname not set and initializes otherwise.
+		:return: String hostname
+		"""
 		try:
-			hostname = socket.gethostname()
-		except:
-			hostname = ERROR.HOSTNAME
-		return hostname
+			return self._hostname
```

Comment:
I don't know either.

Reply:
Before, there was the if else, like this:
```python
if not self._hostname:
    self._hostname = socket.gethostname()

return self._hostname
```

But this gave me an attribute error, because you cannot check with 'if' for an unset variable.